tests/version_test.go

```diff
@@ -10,7 +10,6 @@ var _ = Describe("Version", func() {
 	It("prints its version", func() {
 		output, err := execute("deis --version")
 		Expect(err).NotTo(HaveOccurred())
-		// TODO: read the expected version from ../client/deis-version?
-		Expect(output).To(Equal("2.0.0-dev\n"))
+		Expect(output).To(MatchRegexp("2.0.0-dev-[0-9a-f]{7}\n"))
```

Comment:
Why not do a regex that reads something like:

3 digit, dot separated version
maybe a -dev suffix with a -(git sha with max 7 digits)